### ISSUE TITLE
Global Styles: Pagination buttons must have a tooltip

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#62283]
### Description

Any icon button must visually expose the button accessible name via a tooltip. Worth mentioning tooltips are already a compromise for accessibility, as visible text is always preferred.

The Global styles pagination buttons are an interesting case, as the implementation breaks the logic of the Button component in the first place. To me, this is a sign that the component can be misused. Cc @mirka  

The Button component logic is, shortly:
- When the Button has a text children, it doesn't show a tooltip (unless there's a keyboard shortcut that needs to be shown in the tooltip).
- When the Button only shows an icon, the `lebel` prop is used for the tooltip.

Turns out the Global styles pagination buttons contain text children, but [they're unicode characters for the chevrons](https://github.com/WordPress/gutenberg/blob/a9fd33c99b69ef6ea747fed96569ad837632aedb/packages/edit-site/src/components/pagination/index.js#L46-L54): `«, ‹, ›, »`.

As such, even by changing the buttons `aria-label` to `label`, the tooltip is not rendered because the Button component _thinks_ there is visible text.

Unfortunately, the 'visible' text is just glyphs that are visually equivalent to icons. There is no accessible name exposed.

This is clearly a misuse of the component but it could happen in other scenarios as well, then the 'text' isn't meaningful text that exposes the control's accessible name.

In this specific case, a quick fix would be replacing the `«, ‹, ›, »` with actual icons by the means of the `icon` prop.

### Step-by-step reproduction instructions

- Go to Site editor > Styles > Revisions.
- Make sure there's enough revisions to show the pagination buttons ad the end of the revisions list.
- Hoer or focus the pagination buttons.
- Observe no tooltips appear.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ramonjd** [comment]
Thanks for the issue!

The Button component has a `showTooltip` prop that allows tooltips. I've added it over in https://github.com/WordPress/gutenberg/pull/62395

Would something like that be a decent interim measure? I checked the Icons library and there were no obvious replacements for First and Last. 

There are single chevron icons for Next and Previous however.

The Patterns library originally used the pagination component, however since the migration to data views, it has a dropdown with icons:

<img width="244" alt="Screenshot 2024-06-07 at 3 13 01 PM" src="https://github.com/WordPress/gutenberg/assets/6458278/34dc4eba-f2da-46de-938b-ef3fec111d88">

There's an [issue](https://github.com/WordPress/gutenberg/issues/58353) which alludes to revisions using data views so perhaps it could be looked at as part of that.

**Comment by afercia** [comment]
> The Button component has a showTooltip prop that allows tooltips. 

Ah right, I forgot. Yes I think it can work as an interim solution.

> There are single chevron icons for Next and Previous however.

I will create an issue to propose to add those two icons.

**Comment by afercia** [comment]
Re: Pagination in general, there's also https://github.com/WordPress/gutenberg/issues/58133 to propose to have only _one_ Pagination component instead of duplicate code.

**Comment by afercia** [comment]
Hum double checking [the Icons library in the Storybook](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library), I do see both single chevron and double chevron are available>

![Screenshot 2024-06-07 at 09 17 53](https://github.com/WordPress/gutenberg/assets/1682452/23bc37fc-232e-4458-b479-7ac10ae59622)

![Screenshot 2024-06-07 at 09 18 20](https://github.com/WordPress/gutenberg/assets/1682452/cb7eee18-6607-48c9-a002-97b538c4ef5d)

![Screenshot 2024-06-07 at 09 18 34](https://github.com/WordPress/gutenberg/assets/1682452/37bfd244-4919-4138-824a-620828d4f407)

Note: maybe the 'previous' and 'next' could have used a better name. To me, they're more 'first' and 'last'.

**Comment by ramonjd** [comment]
> Hum double checking [the Icons library in the Storybook](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library), I do see both single chevron and double chevron are available

Nice spotting! I think I just keyed in "chevron".

**Comment by afercia** [comment]
Yea, I did the same first 😆
